### PR TITLE
Horny Mode

### PR DESCRIPTION
**Pull request by ess4654**
Removed Twitter's Trust and Safety rules that derank NSFW and adult content for a totally horny After Dark 18+ mode of Twitter 🍆💦

### CODE REVIEW

**Comment by ess4654** [comment]
I've never coded in scala. Apologies for anything that breaks

**Comment by biraj6116** [comment]
Looks good, approved

**Comment by ANormalProgrammer** [comment]
LGTM

**Comment by Foxlass** [comment]
Merge this ASAP

**Comment by ArchitW** [comment]
flagged!!

**Comment by zarttic** [comment]
👀

**Comment by AndKenneth** [comment]
lgtm

**Review by utkarsh-aryan — COMMENTED** [review]
LGTM

**Comment by erodactyl** [comment]
chad

File: src/java/com/twitter/search/earlybird/common/userupdates/UserUpdatesChecker.java [hunk 1]
```diff
@@ -45,9 +45,27 @@ public boolean skipUserUpdate(UserUpdate userUpdate) {
...
+//⣿⣿⡄⢨⣻⣽⣿⣟⣿⣞⣗⡽⡸⡐⢸⣿⣿⣿⣿⣿⣿⣿
+//⣿⣿⡇⢀⢗⣿⣿⣿⣿⡿⣞⡵⡣⣊⢸⣿⣿⣿⣿⣿⣿⣿
+//⣿⣿⣿⡀⡣⣗⣿⣿⣿⣿⣯⡯⡺⣼⠎⣿⣿⣿⣿⣿⣿⣿
+//⣿⣿⣿⣧⠐⡵⣻⣟⣯⣿⣷⣟⣝⢞⡿⢹⣿⣿⣿⣿⣿⣿
+//⣿⣿⣿⣿⡆⢘⡺⣽⢿⣻⣿⣗⡷⣹⢩⢃⢿⣿⣿⣿⣿⣿
+//⣿⣿⣿⣿⣷⠄⠪⣯⣟⣿⢯⣿⣻⣜⢎⢆⠜⣿⣿⣿⣿⣿
+//⣿⣿⣿⣿⣿⡆⠄⢣⣻⣽⣿⣿⣟⣾⡮⡺⡸⠸⣿⣿⣿⣿
+//⣿⣿⡿⠛⠉⠁⠄⢕⡳⣽⡾⣿⢽⣯⡿⣮⢚⣅⠹⣿⣿⣿
+//⡿⠋⠄⠄⠄⠄⢀⠒⠝⣞⢿⡿⣿⣽⢿⡽⣧⣳⡅⠌⠻⣿
+//⠁⠄⠄⠄⠄⠄⠐⡐⠱⡱⣻⡻⣝⣮⣟⣿⣻⣟⣻⡺⣊
```
Comment (ARXChrono) [inline src/java/com/twitter/search/earlybird/common/userupdates/UserUpdatesChecker.java:68]: Make it smaller for better accuracy.

**Comment by Ansenn** [comment]
LGTM

File: src/java/com/twitter/search/earlybird/common/userupdates/UserUpdatesChecker.java (hunk 1 above)
Comment (StepL7) [reply src/java/com/twitter/search/earlybird/common/userupdates/UserUpdatesChecker.java:68]: 可以画的再抽象点

**Comment by MurphyAdam** [comment]
LGTM

**Comment by moosa817** [comment]
LGTM

**Comment by rzk3** [comment]
LGTM

**Comment by michaelknudsen** [comment]
lgtm

**Comment by h3llo-wor1d** [comment]
LGTM

**Comment by FoxMoss** [comment]
sign the cla we need to see this merged

**Comment by angelonazzaro** [comment]
Man, that's some serious art right there

**Comment by maddsua** [comment]
bruh is that a boomerang?

**Comment by wolfvandenbussche** [comment]
LGTM

**Comment by solarite1797** [comment]
Perfection.

LGTM, Merge this.

**Comment by chikof** [comment]
LGTM

**Comment by rendiputra** [comment]
LGTM

**Comment by zhumqs** [comment]
LGTM.

**Comment by zhumqs** [comment]
Please merge ASAP!

**Pull request closed without merging**